lib/takeaway.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'food_menu.rb'
+require_relative 'order_management.rb'
+require_relative 'twilio_message.rb'
+
+class Takeaway
+
+  attr_reader :food_menu, :order_management, :message
+
+  def initialize(food_menu, order_management = OrderManagement.new, message = Message.new)
+    @food_menu = food_menu
+    @order_management = order_management
+    @message = message
+  end
+
+  def see_dishes
+    food_menu.print
+  end
+
+  def place_the_order(selected_dishes)
+    selected_dishes.each do |dish, number|
```

Comment:
For SRP, you could extract the each loop that adds the orders into a private method and then call this in your place_order method. This would also make the method more readable.